universal-login-sdk/test/authentication-relayer.ts

```diff
@@ -0,0 +1,49 @@
+import {expect} from 'chai';
+import {Wallet, utils} from 'ethers';
+import {createFixtureLoader} from 'ethereum-waffle';
+import basicSDK from './fixtures/basicSDK';
+import UniversalLoginSDK from '../lib/sdk';
+import { CancelAuthorisationRequest } from '@universal-login/commons/lib';
+
+const loadFixture = createFixtureLoader();
+
+describe('E2E authorization - sdk <=> relayer', async () => {
+  let relayer: any;
+  let sdk: UniversalLoginSDK;
+  let contractAddress: any;
+  let privateKey: any;
+
+  beforeEach(async () => {
+    ({sdk, privateKey, contractAddress, relayer} = await loadFixture(basicSDK));
+  });
+
+  afterEach(async () => {
+    await relayer.clearDatabase();
+  });
+
+  it('Send valid cancel request', async () => {
```

Comment:
Shall we make it roundtrip with creating request first? 
Here we cancel non-existing request and everything seems fine.

Reply:
Hmm, the purpose of this test was to check against authorisation.